File: leetcode/100-999/171. Excel Sheet Column Number/solution_1.cs
public class Solution 
{
    public int TitleToNumber(string columnTitle) 
    {
        int title = 0;
        int power = 0;

        for (int i = columnTitle.Length - 1; i >= 0; --i)
        {
            int num = columnTitle[i] - 64;
            title += num * (int) Math.Pow(26, power++);
        }    

        return title;
    }
}

File: leetcode/100-999/171. Excel Sheet Column Number/solution_1.java
class Solution 
{
    public int titleToNumber(String columnTitle) 
    {
        int title = 0;
        int power = 0;

        for (int i = columnTitle.length() - 1; i >= 0; --i)
        {
            int num = columnTitle.charAt(i) - 64;
            title += num * Math.pow(26, power++);
        }    

        return title;
    }
}